packages/main/src/Menu.ts

```diff
@@ -190,6 +195,32 @@ type OpenerStandardListItem = StandardListItem & { associatedItem: MenuItem };
  * @since 1.10.0
  */
 @event("after-close")
+
+/**
+ * Fired when a menu item receives focus. <b>This event does not bubble.</b>
```

Comment:
Let's skip the `<b>This event does not bubble.</b>` part for now - see my next comment

Reply:
done